addons/sale/views/product_views.xml

```diff
@@ -90,15 +90,6 @@
                     </div>
                 </button>
             </div>
-            <group name="description" position="after">
-                <t groups="sales_team.group_sale_salesman">
-                    <group string="Warning when Selling this Product" groups="sale.group_warning_sale">
-                        <field name="sale_line_warn" string="Warning"/>
-                        <field name="sale_line_warn_msg" string="Message" placeholder="Type a message..."
-                                attrs="{'required':[('sale_line_warn','!=','no-message')],'readonly':[('sale_line_warn','=','no-message')], 'invisible':[('sale_line_warn','=','no-message')]}"/>
```

Comment:
Lost the invisible, contradictory with the readonly though, but makes more sense IMHO